src/gluonts/model/deepar/_estimator.py

```diff
@@ -306,71 +305,65 @@ def create_transformation(self) -> Transformation:
         if not self.use_feat_dynamic_real:
             remove_field_names.append(FieldName.FEAT_DYNAMIC_REAL)
 
-        return Chain(
-            [RemoveFields(field_names=remove_field_names)]
-            + (
-                [SetField(output_field=FieldName.FEAT_STATIC_CAT, value=[0.0])]
-                if not self.use_feat_static_cat
-                else []
+        transform = RemoveFields(field_names=remove_field_names)
```

Comment:
Maybe call it "transforms" or explicitly state that it's a `Chain` here? Using `+=` on a single transform seems weird at first sight.